app/src/pages/home.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect, useContext } from 'react';
+
+import { useNavigate } from 'react-router-dom'
+
+import CircularProgress from '@mui/material/CircularProgress';
+import Paper from '@mui/material/Paper';
+import Snackbar from '@mui/material/Snackbar';
+import Button from '@mui/material/Button'
+import Typography from '@mui/material/Typography'
+
+import app from 'STARTER/feathers-client';
+import Login from 'STARTER/components/login';
+import Registration from 'STARTER/components/registration';
+import FormattedMessage from 'STARTER/components/formattedMessage'
+import { AuthContext, AuthContextType, authData } from 'STARTER/contexts/auth'
+
+import { isMobile } from 'STARTER/utils';
+
+export default function Home() {
+	const [snackBarOpen, setSnackBarOpen] = useState(false);
+	const [snackBarMessage, setSnackBarMessage] = useState('');
+
+  const { authenticate, isLoginLoading, isAuthed, logout } = useContext(AuthContext) as AuthContextType
+
+	const handleCloseSnackBar = () => setSnackBarOpen(false);
+  let navigate = useNavigate()
+
+	const onMobile = isMobile();
+
```

Comment:
```suggestion
	useEffect(() => {
		if (isAuthed) navigate('/dashboard', { replace: true })
	}, [isAuthed])

```

This will redirect users to the dashboard if a user is already logged in.

Reply:
this should be included in the latest commits!